rwgame/RWGame.cpp

```diff
@@ -554,6 +555,22 @@ void RWGame::tick(float dt) {
             clockAccumulator -= 1.f;
         }
 
+        if (prevGameHour != state.basic.gameHour) {
```

Comment:
what about making it inline function, so the `tick` doesn't get so huge when all the rules get implemeneted?